### DISCUSSION TITLE
[ 功能建议 ]“实例管理-会话管理”页面建议增加“会话告警”查看功能，活动会话数量超过阈值时记录活动会话快照

### DISCUSSION THREAD

**Discussion by yuandajn578** [hhyo/Archery#1589] [Ideas]
### 目前遇到的问题/使用障碍
实例活动会话过多时，无法及时查看、定位当时的活动会话信息
### 希望如何解决/实现它
“实例管理-会话管理”页面建议增加“会话告警”查看功能，活动会话数量超过阈值时，记录数据库活动会话快照，以便后续处理。

**Comment by hhyo** [comment]
#1583
